pip/pep425tags.py

```diff
@@ -124,7 +124,54 @@ def get_platform():
         split_ver = release.split('.')
         return 'macosx_{0}_{1}_{2}'.format(split_ver[0], split_ver[1], machine)
     # XXX remove distutils dependency
-    return distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    result = distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    if result == "linux_x86_64" and sys.maxsize == 2147483647:
```

Comment:
Fixed in wheel: https://bitbucket.org/pypa/wheel/commits/4da780b849affbff98a0defb21ad1f30a94d6f57